### DISCUSSION TITLE
Are Webhooks Available For Self-Hosted Servers?

### DISCUSSION THREAD

**Discussion by SerajMuftah** [supabase/supabase#12372] [Questions] [answered]
We've hosted our Supabase server on our EC2 instance on AWS using Docker (with Supabase Postgres image). However, when using the studio to navigate through the different components of the database, the webhooks option is no where to be found even though it exists in the database (when you look through the schema using something like pgAdmin for instance). So my question is, since webhooks are still an alpha feature, is it available for self-hosted server or am I doing something wrong?

**Comment by silentworks** [comment, marked as answer]
This should be available but just haven't been added to the menu as yet from what I can see. If you visit the `/project/default/database/hooks` path in the browser then it should show up.

**Reply by SerajMuftah** [reply]
It worked! Thank you so much.

**Comment by ruggi99** [comment]
They have added it to the menu recently, about 2 weeks ago, in this commit https://github.com/supabase/supabase/commit/0df99c2d31136cbd76c1476cc0232ac0433e090b.
But `docker-compose.yml` is outdated.
Try the last version of supabase (check on hub.docker.com, latest tag doesn't work actually).
I tried and I can see the menu entry in sefl-hosted version

**Reply by SerajMuftah** [reply]
Thank you for mentioning that. In case anyone encounters the same issue, just use the studio image version `20230203-d2d26d` in the `docker-compose.yaml` as referenced below:

https://github.com/supabase/supabase/issues/10319#issuecomment-1415185175
